fontbe/src/glyphs.rs

```diff
@@ -176,6 +264,7 @@ fn cubics_to_quadratics(glyph: CheckedGlyph) -> CheckedGlyph {
 enum CheckedGlyph {
     Composite {
         name: GlyphName,
+        components: HashMap<(GlyphName, NormalizedLocation), Affine>,
```

Comment:
should make this a Vec -- otherwise you're only allowing one component with unique glyph name per composite glyph at given location.

E.g. think about glyph for ELLIPSIS, it may well have three components named "period" ...

Reply:
Added a test. I even used multiple period. Now a vec. Fixed.